testing/utils/bpcmp/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+include(ADIOSFunctions)
+
+add_executable(Test.Utils.BPcmp TestUtilsBPcmp.cpp)
+
+target_link_libraries(Test.Utils.BPcmp adios2::c adios2::cxx11)
+set(cmd_executor)
+
+add_test(NAME Utils.BPcmp
+  COMMAND ${cmd_executor} $<TARGET_FILE:Test.Utils.BPcmp>
```

Comment:
what is cmd_execute assigned to

Reply:
To nothing for now, but once we add MPI to bpcmp it will be set to:
```
  set(cmd_executor ${MPIEXEC_COMMAND} ${MPIEXEC_NUMPROC_FLAG} 1)
```
similarly to what is in `../../testing/utils/cwriter/CMakeLists.txt` so I will leave it like this for now.

Reply:
I recommend to remove not used code, we never now how the code evolve in the future. Comments seems better here

Reply:
Done